src/routes/notifications/notifications.controller.ts

```diff
@@ -16,4 +23,14 @@ export class NotificationsController {
   ): Promise<void> {
     return this.notificationsService.registerDevice(registerDeviceDto);
   }
+
+  @ApiOkResponse()
```

Comment:
Why do we need to decorate the route with `@ApiOkResponse()`?

Reply:
No reason, in this case Nest will default to status 200 👍🏻

Reply:
Changed in [d880fcb](https://github.com/5afe/safe-client-gateway-nest/pull/268/commits/d880fcb3732f5bd60894507bcb3104e4a46b7714)